cypress/integration/providers_spec.js

```diff
@@ -0,0 +1,111 @@
+import { shouldBeRedirectedToLogin } from '../support/assertions';
+
+describe('Dashboard Providers Page', () => {
+  describe('When not logged in', () => {
+    it('should redirect to login page', () => {
+      cy.visit('/#/providers');
+      shouldBeRedirectedToLogin();
+    });
+  });
+
+  describe('When logged in', () => {
+    // test provider name
+    const name = 'TESTPROVIDER';
+    beforeEach(() => {
+      cy.login();
+    });
+
+    it('displays a link to view providers', () => {
+      cy.visit('/');
+
+      cy.get('nav li a').contains('Providers').should('exist').as('providers');
+      cy.get('@providers').should('have.attr', 'href', '#/providers');
+      cy.get('@providers').click();
+
+      cy.url().should('include', 'providers');
+      cy.get('.heading--xlarge').should('have.text', 'Providers');
+
+      cy.get('table tbody tr').its('length').should('be.gt', 1);
```

Comment:
I feel like this should test for either > 1 OR exactly the length of our fixture